ocamltest/main.ml

```diff
@@ -151,9 +151,10 @@ let test_file test_filename =
   let test_build_directory_prefix =
     get_test_build_directory_prefix test_directory in
   let clean_test_build_directory () =
-    ignore
-      (Sys.command
-         (Filename.quote_command "rm" ["-rf"; test_build_directory_prefix]))
+    try
+      Sys.rm_rf test_build_directory_prefix
+    with Sys.Break as e -> raise e
+       | _ -> ()
```

Comment:
Could we instead here just catch `Sys_error _ -> ()` and let other exceptions pass through?

Reply:
Yes, that's a better idea, thanks!